docs/configuration/install.md

```diff
@@ -90,11 +90,12 @@ We RECOMMEND at least 100 Mbit/sec. for the web server WAN connection.
 For best results we RECOMMEND:
 
   * Debian GNU Linux 8 / Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
-  * MySQL 5.5 / MariaDB
-  * PHP 5.6+
-  * Apache 2.2+ with mod_php
+  * MySQL 5.6+
+  * MariaDB 10+
```

Comment:
I would prefer to get rid of "open end version" recommendations completely. E.g. instead of **MariaDB 10+** I would prefer **MariaDB 10.0 - 10.2** or **MariaDB 10.1**. If in doubt I would always use the exact version of our test system.

Reply:
+1